cogboard-webapp/src/actions/thunks.js

```diff
@@ -47,6 +50,12 @@ export const fetchInitialData = () => dispatch => {
   }, console.error);
 };
 
+export const fetchAppInfo = () => dispatch => {
+  return fetchData(URL.LOAD_INFO).then(_ =>
```

Comment:
You don't need parameter here, please add also error handler, but without catch method. As a reject callback.

Reply:
Fixed